app/controllers/csvs_controller.rb

```diff
@@ -41,7 +40,7 @@ def download
         send_file @csv_export.path_file, type: :csv, filename: @csv_export.download_name
         @csv_export.status! :downloaded
         Rails.logger.info("#{current_user.name_and_email} just downloaded #{@csv_export.download_name})")
-      rescue
+      rescue ActionController::MissingFile
         @csv_export.status! :deleted
```

Comment:
actually, won't this codepath be hit if a request comes in the small window of time if a file is `pending` and hasn't been created yet by the worker?